typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -99,85 +108,129 @@ export class EvmIsmModule extends HyperlaneModule<
   ): Promise<AnnotatedEV5Transaction[]> {
     targetConfig = IsmConfigSchema.parse(targetConfig);
 
-    // Do not support updating to a custom ISM address
-    if (typeof targetConfig === 'string') {
-      throw new Error(
-        'Invalid targetConfig: Updating to a custom ISM address is not supported. Please provide a valid ISM configuration.',
-      );
+    // Nothing to do if its the default ism
+    if (typeof targetConfig === 'string' && isZeroishAddress(targetConfig)) {
+      return [];
```

Comment:
Does it mean we now allow changing the IGP to a plain address? This caused an error before

Reply:
The IGP is a hook, here we are working with ISMs but yeah we are going to allow setting the ISM to a custom address as there is no real limitation to why this shouldn't be allowed.  Initially we did not allow setting to a custom address in hooks too but we changed it so this would bring feature parity to ISMs too